src/coreclr/pal/src/exception/signal.cpp

```diff
@@ -190,13 +190,20 @@ BOOL SEHInitializeSignals(CorUnix::CPalThread *pthrCurrent, DWORD flags)
         handle_signal(SIGSEGV, sigsegv_handler, &g_previous_sigsegv);
 #else
         handle_signal(SIGTRAP, sigtrap_handler, &g_previous_sigtrap);
+        int additionalFlagsForSigSegv = 0;
+#ifndef __sun
```

Comment:
nit: `HOST_SUNOS` (here and other places) which is a prevalent pattern in this file.